README.md

```diff
@@ -1,2 +1,222 @@
 # stellar-dbt-public
 Public DBT instance to aid in data transformation for analytics purposes
+
+## Table of Contents
+- [Data Transformation](#data-transformation)
+    - [Workflow](#workflow)
+    - [dbt Project Structure](#dbt-project-structure)
+    - [Tests](#tests)
+- [Getting Started](#getting-started)
+    - [Oauth connection](#oauth)
+- [dbt Setup](#dbt-setup)    
+- [Working with dbt](#working-with-dbt)
+    - [Running Models](#running-models)
+    - [Running Tests](#running-tests)
+- [Project Structure](#project-structure)
+    - [Development Folders](#development-folders)
+
+
+## Data Transformation 
+
+dbt is composed of different moving parts working harmoniously. All of them are important to what dbt does — transforming data.  When you execute dbt run, you are running a model that will transform your data without that data ever leaving your warehouse.
+
+### Workflow
+
+The top level of a dbt workflow is the project. A project is a directory of a `.yml` file (the project configuration) and either `.sql` or `.py` files (the models). The project file tells dbt the project context, and the models let dbt know how to build a specific data set.
+
+A model is a single file containing a final select statement, and a project can have multiple models, and models can even reference each other.
```

Comment:
Might be worthwhile to mention that the end result of a model is a data mart table, with the intent of simplifying analytics.